### PR TITLE
fix: add sdk autoupdates

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

Automatically update sdk go. 
Fix issue with version bumps

### CODE REVIEW

File: .github/workflows/update-sdk.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        uses: tj-actions/verify-changed-files@78dc414e915e0664bcf0d2b42465a86cd47bcc3c
+        id: verify-changed-files
+        with:
+          files: |
+             ./internal/**/*
+      - uses: peter-evans/create-pull-request@v5
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+
+        with:
+          title: "fix: atlas GO sdk update"
```
Comment (AgustinBettati) [inline .github/workflows/update-sdk.yml]: would define this with `chore:` prefix as we have for dependabot

File: scripts/update-sdk.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+LATEST_SDK_RELEASE=$(curl -sSfL -X GET  https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name' | cut -d '.' -f 1)
+echo  "==> Updating SDK to latest major version $LATEST_SDK_RELEASE"
+gomajor get "go.mongodb.org/atlas-sdk/$LATEST_SDK_RELEASE@latest"
+go mod tidy
+echo "Done, remember to update build/ci/library_owners.json"
```
Comment (AgustinBettati) [inline scripts/update-sdk.sh]: did not understand this finishing echo, is there any manual changes needed before merging the generated PR?

**Review by AgustinBettati — APPROVED** [review]
Great change, comment with small details

File: .github/workflows/update-sdk.yml (hunk 1 above)
Comment (wtrocki) [reply .github/workflows/update-sdk.yml]:
```suggestion
          title: "chore: atlas GO sdk update"
```

File: scripts/update-sdk.sh (hunk 2 above)
Comment (wtrocki) [reply scripts/update-sdk.sh]:
```suggestion
echo "Finished update"
```

Comment (wtrocki) [reply scripts/update-sdk.sh]: Removed

File: .github/workflows/update-sdk.yml (hunk 1 above)
Comment (wtrocki) [reply .github/workflows/update-sdk.yml]: Done

**Review by AgustinBettati — APPROVED** [review]

**Pull request merged**